tests/admin_inlines/tests.py

```diff
@@ -175,6 +175,21 @@ def test_help_text(self):
             1
         )
 
+    def test_tabular_help_text_readonly_field(self):
+        response = self.client.get(reverse('admin:admin_inlines_someparentmodel_add'))
+        self.assertContains(
+            response,
+            '<img src="/static/admin/img/icon-unknown.svg" '
+            'class="help help-tooltip" width="10" height="10" '
+            'alt="(Test help text)" '
+            'title="Test help text">',
+            1
+        )
+
+    def test_tabular_label_readonly_field(self):
+        response = self.client.get(reverse('admin:admin_inlines_someparentmodel_add'))
+        self.assertContains(response, '<th>Test label</th>', html=True)
+
```

Comment:
Can you combine these two tests into one and add an explanatory docstring. See `test_help_text` directly above for an example.

Something like: 

```
"""
Tabular inlines use help_text and label when defined on ModelForm.Meta. (#29184)
"""
```

This makes it clear what the test is for.